Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -77,28 +77,28 @@ public LayerMask[] MagneticSurfaces
 
         [SerializeField]
         [Tooltip("Max distance for raycast to check for surfaces")]
-        private float maxDistance = 3.0f;
+        private float maxRaycastDistance = 50.0f;
 
         /// <summary>
         /// Max distance for raycast to check for surfaces
         /// </summary>
-        public float MaxDistance
+        public float MaxRaycastDistance
         {
-            get { return maxDistance; }
-            set { maxDistance = value; }
+            get { return maxRaycastDistance; }
+            set { maxRaycastDistance = value; }
         }
 
         [SerializeField]
         [Tooltip("Closest distance to bring object")]
-        private float closeDistance = 0.5f;
+        private float closestDistance = 0.5f;
 
         /// <summary>
         /// Closest distance to bring object
         /// </summary>
-        public float CloseDistance
+        public float ClosestDistance
```

Comment:
>ClosestDistance [](start = 21, length = 15)

Please add a note on Solver migration in GA here:

[https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/UpdatingToGA.html](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/UpdatingToGA.html)